doc/source/whatsnew/v0.17.0.txt

```diff
@@ -46,12 +46,51 @@ Backwards incompatible API changes
 
 .. _whatsnew_0170.api_breaking.other:
 
+Changes to convert_objects
+^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
you can add a link name, eg ``.. _whatsnew_0170.api_breaking.convert_objects:`` to refer to